tutorials/clustering.md

```diff
@@ -13,48 +12,53 @@ In this tutorial, we will show how you can create a distributed hypertable withi
 a distributed database. We will set up the distributed database with one primary access node
 and two data nodes across three distinct TimescaleDB instances.
 
-Prerequisites:
-- 3 instances of TimescaleDB installed (private BETA branch only)
-- A superuser set up on all 3 instances of TimescaleDB
-- Network connectivity between the 3 TimescaleDB instances
-- Must be on PG11
+It is assumed before starting this tutorial you will have three networked machines set up with:
+- TimescaleDB version 2.0 (beta) or greater
+- PostgresSQL 11
+
+For the sake of this tutorial we will refer to these machines as
+access.example.com, data1.example.com, data2.example.com. You will also
+need a common superuser role on each postgres instance. This tutorial
+will assume the `postgres` role for this, but you may substitute any
+role common to all machines with superuser permission.
```

Comment:
Perfect, but I suggest to put backquotes around the domain names.

Reply:
done

Reply:
done